src/swupd/README.vendor

```diff
@@ -0,0 +1,25 @@
+This package uses govendor tool for "vendoring" packages,
```

Comment:
Is "govendor" the right tool. The tool that is almost certainly going to be the eventual solution is [go dep](https://github.com/golang/dep)

Reply:
govendor is sufficient. Not sure if it is "the right one". I'm not familiar with go dep, other than "when it is ready to use, it will be merged into go". I can take a look at it and follow up.

Reply:
Following up: I read on go dep, and was convinced that we can start using it even if it is not merged to go tool yet. The workflow is similar to govendor for our use cases. I've updated the commits that add vendor code and the README.vendor.